### ISSUE TITLE
Ability to run Cotonti from the command line.

### ISSUE THREAD

**Issue by Alex300** [Cotonti/Cotonti#1698]
Ability to run Cotonti from the command line.

We need to be able to run the application with the CLI. Useful, for example, to run cron tasks.

[RU]
Нужна возможность запускать Cotonti из командной строки. Это полезно, например, для выполнения Cron задач.
Запуск должен работать примерно так:

```
php /path/to/app/cot.php cleaner/clean --scope=trash
```

где:
- `/path/to/app/` - путь к скрипту
- `cleaner/clean` - расширение и действие
- `--scope=trash` - параметры

Хорошо бы еще при вызове `php /path/to/app/cot.php help` выводить справку с возможным списком команд

**Comment by Dayver** [comment]
А кроме крона есть другие сценарии использования?

**Comment by Alex300** [comment]
Это в основном служебные задачи. Крон самая распространенная, и крон как раз и запускает процессы в CLI. Есть еще такие как конвертация данных, экспорт или копирование больших объемов данных. Командная строка доступна только администраторам сервера и недоступна обычным пользователям. При этом она не имеет ограничений, свойственных веб серверу, например время исполнения скрипта или память.

Есть и другие задачи - постоянно висящие процессы deamon's, web-сокеты - но об этом пока рано говорить.

**Comment by Dayver** [comment]
Точно, вебсокеты, совсем забыл про них. Сам недавно выдумывал велосипед с ними.

Если с кроном можно найти компромисс то с сокетами действительно это нужно
